Documentation/contributing.rst

```diff
@@ -55,6 +55,39 @@ requirements have been met:
 * You have signed off on your commits, see the section "Developer's
   Certificate of Origin" for more details.
 
+Release Process
+---------------
+
+Cilium schedules a major release every 3 months. Each major release is
+performed by incrementing the `Y` in the version format `X.Y.0`. The group of
+committers can decide to increment `X` instead to mark major milestones in
+which case `Y` is reset to 0.
+
+The following steps are performed to publish a release:
+
+1. The master branch is set to the version `X.Y.90` at all times. This ensures
```

Comment:
Why `.90`? Can't we just append a `+`? F.e. release commit is `X.Y.0`, next development cycles is `X.Y.0+` until it's newly released as `X.[Y+1].0`. And the `+` is only ever used in devel branch, never stable.